doc/man1/flux-mini.rst

```diff
@@ -825,6 +834,56 @@ overridden in some cases:
    Copy files previously mapped with :man1:`flux-filemap` to $FLUX_JOB_TMPDIR.
    See :man1:`flux-shell` for more *stage-in* options.
 
+.. _submission_directives:
+
+SUBMISSION DIRECTIVES
+=====================
+
+The *flux mini batch* command supports submission directives
+mixed within the submission script. The submission directive specification
+is fully detailed in RFC 36, but is summarized here for convenience:
+
+ * A submission directive is indicated by a line that starts with
+   a prefix of non-alphanumeric characters followed by a tag ``FLUX:`` or
+   ``flux:``. The prefix plus tag is called the *directive sentinel*. E.g.,
+   in the example below the sentinel is ``# flux:``: ::
+
+     #!/bin/sh
+     # flux: -N4 -n16
+     flux mini run -n16 hostname
+
+ * All directives in a file must use the same sentinel pattern, otherwise
+   an error will be raised.
+ * Directives must be grouped together - it is an error to include a
+   directive after any non-blank line that doesn't start with the common
+   prefix.
+ * The directive starts after the sentinel to the end of the line.
+ * The ``#`` character is supported as a comment character.
```

Comment:
Might be good to add "in directives" to the comment character bullet (so people don't think they can prefix the sentinel with a comment)

Reply:
Great suggestion!